apps/onesky-app/app/global.css

```
@@ -3,502 +3,506 @@
+  background-color: #ffffff;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  height: 9%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  border-top: #d9ebb2 1px solid;
+  padding-top: 0;
+  padding-bottom: 3%;
+  padding: 3%;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)